### PR TITLE
Add organisation tag actions and tests.

### PR DESCRIPTION
**Pull request by reiterl**
Don't include permissions. Wait for https://github.com/OpenSlides/openslides-backend/pull/640 for this.
Orga level permissions should be easy then.

### CODE REVIEW

**Review by jsangmeister — APPROVED** [review]
Please note the actions mentioned in #639 that can set organisation tags. If you do not want to implement that in this PR, then please open a new issue for that. Same goes for the permissions.

**Review by FinnStutzenstein — COMMENTED** [review]
On create I've added `organisation_id: Id;` to the payload. The orga tags weren't assigned to the organisation beforehand.

**Comment by jsangmeister** [comment]
> On create I've added `organisation_id: Id;` to the payload. The orga tags weren't assigned to the organisation beforehand.

Since we only allow one organisation, can't we add that implicitely in the backend? Seems unnecessary that the client always sends it if the only allowed value is 1 anyway.

**Comment by FinnStutzenstein** [comment]
If we add more orgas, this is one place less to change and one special case less to program into since just adding `organisation_id` should work out of the box. The current limitation is the client, not the backend. Yes, there are no `organisation.create` actions, but technically, this would work.

**Comment by jsangmeister** [comment]
Ok, if you mean

**Comment by reiterl** [comment]
Add the permissions and test.

**Comment by reiterl** [comment]
Add organisation_id to organisation_tag create.

File: tests/system/action/organisation_tag/test_create.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    def test_create(self) -> None:
+        self.set_models({"organisation/1": {}})
+        response = self.request(
+            "organisation_tag.create",
+            {"name": "wSvQHymN", "color": "#eeeeee", "organisation_id": 1},
+        )
+        self.assert_status_code(response, 200)
+        organisation_tag = self.get_model("organisation_tag/1")
+        assert organisation_tag.get("name") == "wSvQHymN"
+        assert organisation_tag.get("color") == "#eeeeee"
```
Comment (FinnStutzenstein) [inline tests/system/action/organisation_tag/test_create.py:14]: assert, that it is linked to the organisation. check if `organisation/1/organisation_tag_ids` is `[]`.

**Review by FinnStutzenstein — APPROVED** [review]

**Comment by reiterl** [comment]
Add issue https://github.com/OpenSlides/openslides-backend/issues/699 for the set organisation_tag code.

**Pull request merged**